.changelog/2478.txt

```diff
@@ -0,0 +1,5 @@
+```release-note:bug
+api-gateway: fixes bug where envoy will silently reject RSA keys less than 2048 bits in length when not in FIPS mode, and
```

Comment:
I don't have a strong opinion but think it's worth asking: is this a bug, or is it an improvement?

Reply:
I was on the fence here, I went with bug because to end users it would seem like a bug that they provided a certificate but it just wasn't working (unless they knew to dive into the envoy config and parse the output there).